### PR TITLE
CIV-3633: You can't use this service - This service is currently for claimants representing themselves

### PR DESCRIPTION
**Pull request by hmcts-version1-pablo**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-3633

### Change description ###

Another variant of You can't use this service screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -734,6 +734,14 @@
       "N1_PAPER_FORM": "N1 paper form",
       "OPENS_IN_A_NEW_WINDOW": "(opens in a new window)",
       "PAGE_TITLE": "You can’t use this service",
+      "CLAIM_ON_BEHALF": {
+        "CURRENTLY_SERVICE" : "This service is currently for claimants representing themselves.",
+        "LEGAL_REPRESENTATIVE" : "If you’re a legal representative ",
+        "MCOL_LINK": "use the Money Claim Online (MCOL) service",
+        "OR": " or ",
+        "DOWNLOAD_PAPER_LINK": "download a paper form (opens in a new window)",
+        "COMPLETE_RETURN": ", complete and return it to make your claim."
```
Comment (hmcts-version1-domini) [inline src/main/modules/i18n/locales/cy.json:743]: Line 737 "CLAIM_ON_BEHALF": { and then inside, the lines 742 and 743 are out of alphabetical order.

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -733,6 +733,14 @@
...
       "OPENS_IN_A_NEW_WINDOW": "(opens in a new window)",
       "PAGE_TITLE": "You can’t use this service",
+      "CLAIM_ON_BEHALF": {
+        "CURRENTLY_SERVICE" : "This service is currently for claimants representing themselves.",
+        "LEGAL_REPRESENTATIVE" : "If you’re a legal representative ",
+        "MCOL_LINK": "use the Money Claim Online (MCOL) service",
+        "OR": " or ",
+        "DOWNLOAD_PAPER_LINK": "download a paper form (opens in a new window)",
+        "COMPLETE_RETURN": ", complete and return it to make your claim."
+      },
```
Comment (hmcts-version1-domini) [inline src/main/modules/i18n/locales/en.json:743]: Same here. Alphabetic order.

**Review by laperla11 — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (hmcts-version1-pablo) [reply src/main/modules/i18n/locales/cy.json]: Done

File: src/main/modules/i18n/locales/en.json (hunk 2 above)
Comment (hmcts-version1-pablo) [reply src/main/modules/i18n/locales/en.json]: Done

**Review by hmcts-version1-domini — APPROVED** [review]

**Pull request merged**